### ISSUE TITLE
Raw string interpolation doesn't count as accessing variable

### ISSUE THREAD

**Issue by vberlier** [microsoft/pyright#1830]
With the latest release Pyright reports an error in the following code:

```python
def foo():
    message = "hello"  # Variable "message" is not accessed
    print(fr"{message}")
```

**Comment by erictraut** [comment]
Thanks for the bug report. This regression was caused by a recent optimization. I've fixed the problem and added some tests to prevent future regressions for "fr" strings.

**Comment by erictraut** [comment]
This is fixed in 1.1.138, which I just published.
